lib/beaker-hostgenerator/data.rb

```diff
@@ -29,6 +56,873 @@ def pe_upgrade_family
       'm' => 'master',
     }
 
+    BASE_CONFIG = {
+      'HOSTS' => {},
+      'CONFIG' => {
+        'nfs_server' => 'none',
+        'consoleport' => 443,
+      }
+    }
+
+    def base_host_config
+      {
+        'pe_dir' => pe_dir(pe_version, pe_family),
+        'pe_ver' => pe_version,
+        'pe_upgrade_dir' => pe_dir(pe_upgrade_version, pe_upgrade_family),
+        'pe_upgrade_ver' => pe_upgrade_version,
+      }
+    end
+
+    def osinfo
+      {
```

Comment:
I agree, moving it to a more readily-accessible location for other projects would be nice. Isn't that one of the goals of CI.Next Phase 5?